src/it/java/com/google/checkstyle/test/chapter5naming/rule528typevariablenames/MethodTypeParameterNameTest.java

```diff
@@ -38,7 +38,7 @@ protected String getPackageLocation() {
         return "com/google/checkstyle/test/chapter5naming/rule528typevariablenames";
     }
 
-    @BeforeAll
+    @BeforeEach
     public static void setConfigurationBuilder() throws CheckstyleException {
         format = getModuleConfig("ClassTypeParameterName").getProperty("format");
```

Comment:
I feel this before can just be completely removed and inlined.

Reply:
It throws an exception, better to be instantiated in the method.

Reply:
What is the exception?